addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -81,14 +81,34 @@ $-editor-messages-margin-x: 2%;
     }
 
     &.oe_structure.oe_empty, &[data-oe-type=html], .oe_structure.oe_empty {
-        &#wrap:empty, &#wrap > .oe_drop_zone.oe_insert:not(.oe_vertical):only-child {
+        &#wrap:empty, &#wrap > .oe_drop_zone.oe_insert:not(.oe_vertical) {
             @extend %o-editor-messages;
-            padding: 112px 0px;
+
+            &:only-child {
+                padding: 112px 0px;
+            }
+
+            &:not(:only-child) {
+                padding: 0px;
+                &::before {
+                    font-size: 18px;
+                    line-height: 25px!important;
+                }
+            }
```

Comment:
- This code has been copy-pasted (with a nice line break that there should be here too :wink:), can't we share it? Especially since I don't see why there is an exception for `#wrap` anymore?
- Is the padding 0 useful? (just curious)

Reply:
Yes, it's actually to reset the padding which we override in `%o-editor-messages` which was initially 0 before that mixin kicks off: https://github.com/odoo-dev/odoo/blame/6840c928962deb8f9fb870528f2f8749c598a8d0/addons/web_editor/static/src/scss/wysiwyg_snippets.scss#L1954

But I have refactored that part (and will continue later) anyway, instead of erasing for all cases and restoring it in some cases only, I'll just replace it for the cases we want